test/unit.test.js

```diff
@@ -0,0 +1,109 @@
+import { 
+    getServiceClientName, 
+    getServiceClientDescription, 
+    getClientClassName, 
+    getFunctionName, 
+    getFunctionDetails, 
+    getSendOperations, 
+    getReceiveOperations 
+} from '../components/helpers/utils'; 
+
+describe('Service Client Helpers', () => {
+    const mockAsyncAPI = {
+        info: jest.fn(() => ({
+            title: jest.fn(() => 'TestService'),
+            description: jest.fn(() => 'Test Service Description'),
+            hasDescription: jest.fn(() => true)
+        })),
+    };
+
+    const mockAsyncAPIWithoutDesc = {
+        info: jest.fn(() => ({
+            title: jest.fn(() => 'TestService'),
+            description: jest.fn(() => undefined),
+            hasDescription: jest.fn(() => false)
+        })),
+    };
+
+    const mockOperations = [
+        { 
+            operationId: jest.fn(() => 'sendTestOperation'), 
+            id: jest.fn(() => 'receiveTestOperation'),
+            channels: jest.fn(() => [{ address: jest.fn(() => 'test/topic') }]), 
+            summary: jest.fn(() => 'Test Operation Summary'),
+            isSend: jest.fn(() => true),
+            isReceive: jest.fn(() => false),
+        },
+        {
+            operationId: jest.fn(() => 'receiveTestOperation'),
+            id: jest.fn(() => 'receiveTestOperation'),
+            channels: jest.fn(() => [{ address: jest.fn(() => 'test/topic') }]),
+            summary: jest.fn(() => 'Test Operation Summary'),
+            isSend: jest.fn(() => false),
+            isReceive: jest.fn(() => true),
+        }
+    ];
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('getServiceClientName', () => {
+        it('should return the correct service client name', () => {
+            const result = getServiceClientName(mockAsyncAPI);
+            expect(result).toEqual('TestService Client');
+        });
+    });
+
+    describe('getServiceClientDescription', () => {
+        it('should return the correct service client description', () => {
+            const result = getServiceClientDescription(mockAsyncAPI);
+            expect(result).toEqual('Test Service Description');
+        });
+
+        it('should return empty string if no description provided', () => {
+            const result = getServiceClientDescription(mockAsyncAPIWithoutDesc);
+            expect(result).toEqual('');
+        });
+    });
+
+    describe('getClientClassName', () => {
+        it('should return the correct client class name', () => {
+            const result = getClientClassName(mockAsyncAPI);
```

Comment:
test on a mock where title contains spaces, like `Test Title With Space` and you expect `TestTitleWithSpaceClient`